Exercise_4/docs/part-1.md

```diff
@@ -0,0 +1,330 @@
+
+* Unstructured data, typically taking the form of encounter notes, audio, video, or image files, has its own set of significant challenges, and will be out of scope for this tutorial.
+* Structured data, in-contrast, is, as the name suggests, restricted to heavily formatted data types and structures. This could include text fields like name, regular-expressions, like phone number, and alpha-numeric coding terminology. 
+
+Commonly used coding standards for structured data include:  **LOINC, CPT, SNOMED CT, ICD-10, and RxNorm, etc.**​ It is important to have a general understanding of these coding systems because they have become the standard for rendering structured data in an interoperable manner.
+
+### LOINC
+
+**Logical Observation Identifiers Names and Codes (LOINC®)**, is a proprietary clinical terminology that is important for laboratory test orders and results, and is one of a suite of designated standards for use in U.S. Federal Government systems for the electronic exchange of clinical health information. 
+
+Managed by the Regenstrief Institute, LOINC has been identified by the Health Level 7 (HL7) Standards Development Organization as a preferred diagnostic report code set for laboratory test identification in transactions between health care facilities, laboratories, laboratory testing devices and public health authorities.
+
+See here: [https://www.hl7.org/fhir/diagnosticreport-definitions.html#DiagnosticReport.code](https://www.hl7.org/fhir/diagnosticreport-definitions.html#DiagnosticReport.code)
+
+
+LOINC codes are dvidied into Test Order and Test Result codes.
+
+??? abstract "Test Order Code Structure and Examples"
+    Test Order codes are 5 digit numeric codes appended with a dash and 6th number. Form: XXXXX-X
+
+    e.g.,
+        As an example, one of the more popular codes is: `2160-0` for `Creatinine [Mass/volume] in Serum or Plasma`
+        Source:  [https://loinc.org/2160-0/](https://loinc.org/2160-0/)
+
+    Additional attributes associated with LOINC test codes are:
+
+    * Test ID
+    * Test Order Name
+    * Order LOINC Code
+    * LOINC Attributes
+    * Method name
+
+
+    e.g.,
+    An example can be found here: [https://loinc.org/2160-0/](https://loinc.org/2160-0/) and [https://loinc.org/kb/users-guide/introduction/](https://loinc.org/kb/users-guide/introduction/)
+
+
+    | Test ID | Test Order Name | Order LOINC Code | LOINC Attributes | Method Name |
+    |---------|-----------------|------------------|------------------|-------------|
+    | 10AIH  | Factor X Inhibitor Profile, Professional Interpretation | 69049-5 | Coagulation specialist review:Imp:Pt:XXX:Nar: | Only orderable as a reflex. For more information see 10INE / Factor X Inhibitor Evaluation, Plasma. Medical Interpretation |
```

Comment:
The example code (2160-0) is different than what's in the table (69049-5) which is confusing.